docs/quick-start.md

```diff
@@ -272,7 +272,7 @@ scalaVersion := "{{site.SCALA_VERSION}}"
 libraryDependencies += "org.apache.spark" %% "spark-core" % "{{site.SPARK_VERSION}}"
 {% endhighlight %}
 
-For sbt to work correctly, we'll need to layout `SimpleApp.scala` and `simple.sbt`
+For SBT to work correctly, we'll need to layout `SimpleApp.scala` and `simple.sbt`
```

Comment:
I've switched all usages of "SBT" back to "sbt". At one point in time, SBT stood for "Simple Build Tool," but I can't find any remaining evidence of that on their site, so I'm guessing they've moved away from that.